bundles/nim/init.moon

```diff
@@ -0,0 +1,36 @@
+-- Copyright 2012-2015 The Howl Developers
+-- License: MIT (see LICENSE.md at the top-level directory of the distribution)
+
+mode_reg =
+  name: 'nim'
+  extensions: 'nim'
+  create: bundle_load('nim_mode')
+
+howl.mode.register mode_reg
+
+howl.command.register
+  name: 'nim-run'
+  description: 'Compile and run the current Nim file'
+  handler: ->
+    unless howl.app.editor.buffer.file
+      error "No file associated with current buffer."
+
+    exe = howl.app.editor.buffer.config.nim_executable or 'nim'
+    unless howl.io.File(exe).exists
```

Comment:
This won't work when exe is "nim". Either forego the check completely or add a path search for finding the full path. It's a shame to lose the informational warning/error pointing to the correct configuration variable though if we just go with skipping the check.